requirements-dev.txt

```diff
@@ -4,7 +4,7 @@ ipython~=9.1
 ipywidgets>=8.0.0
 packaging>=20.0
 pyink~=24.0
-pyspark-client~=4.0.0
+pyspark[connect]~=4.0.0
```

Comment:
Why do we need this version of pyspark which includes all the JVM jars instead of pyspark-client which is a python only spark connect client?